src/ingestors/index.ts

```diff
@@ -1,7 +1,16 @@
 import { FxHashIngestor } from './fxhash';
 import { MintIngestor } from '../lib/types/mint-ingestor';
 import { ProhibitionDailyIngestor } from './prohibition-daily';
+<<<<<<< HEAD
```

Comment:
It looks like this may have some git merge conflicts

Reply:
yes, will take care of that.